### PR TITLE
i18n support in devconsole Project and QuickStart components

### PR DESCRIPTION
**Pull request by vikram-raj**
fixes: https://issues.redhat.com/browse/ODC-5104

### CODE REVIEW

**Review by rottencandy — APPROVED** [review]
/lgtm

**Comment by rottencandy** [comment]
@vikram-raj needs rebase.

**Review by rottencandy — APPROVED** [review]
/lgtm

**Review by rottencandy — APPROVED** [review]
/lgtm

**Review by rottencandy — APPROVED** [review]
/lgtm

**Comment by jerolimov** [comment]
/title i18n support in devconsole Project and QuickStart components
/retest

File: frontend/packages/dev-console/locales/en/devconsole.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+  "Add": "Add",
```
Comment (jerolimov) [inline frontend/packages/dev-console/locales/en/devconsole.json:2]: Duplicated key in this file, thats the reason why the test still fails

**Comment by jerolimov** [comment]
/retitle i18n support in devconsole Project and QuickStart components

**Review by jerolimov — APPROVED** [review]

File: frontend/packages/dev-console/locales/en/devconsole.json (hunk 1 above)
Comment (vikram-raj) [reply frontend/packages/dev-console/locales/en/devconsole.json:2]: Ah yes, Thank you @jerolimov

**Pull request merged**